lib/worker_manager_test.go

```diff
@@ -0,0 +1,272 @@
+package lib
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/benbjohnson/clock"
+	"github.com/gavv/monotime"
+	"github.com/hanfei1991/microcosm/pkg/p2p"
+	"github.com/stretchr/testify/require"
+)
+
+func TestHeartBeatPingPongAfterCreateWorker(t *testing.T) {
```

Comment:
Can these test cases run in parallel mode

Reply:
Yes